docs/api/tokens/get-non-fungible-token-holdings-tx-metadata.example.schema.json

```diff
@@ -9,6 +9,7 @@
 				"hex": "0x0100000000000000000000000000000803",
 				"repr": "u2051"
 			},
+			"block_height": 38749,
```

Comment:
Make sure these heights match the `block_height` specified below inside `tx` for each example